obspy/clients/fdsn/client.py

```diff
@@ -368,15 +367,12 @@ def _resolve_eida_token(self, token):
         # retrieve user/password using the token
         if self.debug:
             print('Downloading {} with eida token data in POST'.format(url))
-        response = requests.post(url, data=token)
-        # if credentials were returned the status code seems to be 200
-        if response.status_code != 200:
-            msg = ("Failed to resolve EIDA token from URL '{}', server "
-                   "replied with HTTP status code '{!s}' and message "
-                   "'{}'.").format(url, response.status_code,
-                                   response.reason)
-            raise FDSNException(msg)
-        user, password = response.content.decode().split(':')
+
+        # Already does the error checking with fdsnws semantics.
+        response = self._download(url=url, data=token.encode(),
+                                  use_gzip=True, return_string=True)
```

Comment:
Ah, yeah.. didn't use it earlier because this was done pretty early in `__init__` initially and `self._download` couldn't be used yet.